### DISCUSSION TITLE
Why not LICENSE.txt?

### DISCUSSION THREAD

**Discussion by Initial-heart-1** [community/community#35111] [Other Feature Feedback, Questions, & Ideas] [answered]
As the question suggests, we usually use `LICENSE` or `LICENSE.md` as the file name for a licence. When using `LICENSE` as a filename, why not use `LICENSE.txt` to make it easily readable by a text editor?
I tried to find the answer in the Internet, but failed: I couldn't find any information about why this was the case.

**Comment by SilasRodrigues19** [comment]
@Initial-heart-1 U can use txt as the filename, it works too.

![image](https://user-images.githubusercontent.com/47532703/193951830-858c20de-58f4-407c-837e-b51f80f5c9a2.png)

**Reply by Initial-heart-1** [reply]
Of course, I could have chosen to do that. After testing it works. But what I want to know is why people don't usually use `LICENSE.txt`.

**Reply by SilasRodrigues19** [reply]
@Initial-heart-1 I use Material Icon Theme extension which just typing ```LICENSE``` changes the icon so I got used to doing this without any filename

**Comment by jfullstackdev** [comment, marked as answer]
take a look at this 

https://askubuntu.com/questions/1188032/why-do-some-applications-have-files-with-no-extension

https://www.gnu.org/prep/standards/html_node/Releases.html#Releases

we all know, most developers use Linux or macOS which are UNIX-like OSs, and those practices persisted up to these days

even the Linux repo follows it, as you can see here, the README does not have a filename extension

![image](https://user-images.githubusercontent.com/47092464/194097624-c7f59255-a4b8-4263-a8bf-3d379ae49d7f.png)

even their licenses do not have filename extensions

![image](https://user-images.githubusercontent.com/47092464/194097692-b85b8be3-d73a-45a3-b3b5-725681e313ce.png)

https://github.com/torvalds/linux

**Reply by Initial-heart-1** [reply]
I think you are correct, thanks!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Initial-heart-1! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by Initial-heart-1** [comment]
OK，I'm sorry I wasn't aware of this before.




------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;; 
Date: 2022年10月6日(星期四) 晚上9:59
To: ***@***.***&gt;; 
Cc: "Initial ***@***.***&gt;; ***@***.***&gt;; 
Subject: Re: [community/community] Why not LICENSE.txt? (Discussion #35111)





 
Thanks for posting in the GitHub Community, @Initial-heart-1! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
